scrapers/GirlsRimming.yml

```diff
@@ -0,0 +1,28 @@
+name: "GirlsRimming"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - girlsrimming.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[@class='updatesBlock']/h2[@class='title']/text()
+      Details: //meta[@name='description']/@content
+      Image: //div[@class='player-thumb']/img/@src0_1x
+      Studio:
+        Name:
+          fixed: GirlsRimming
+      Movies:
+        Name:
+          selector: //div[@class='updatesBlock']/h2[@class='title']/text()
+          postProcess:
+            - replace:
+              - regex: (.+)(?:\sEp\d:\s).+
```

Comment:
In some series like `https://www.girlsrimming.com/tour/dvds/yoga-session.html` or `https://www.girlsrimming.com/tour/dvds/fidelity.html` there is no episode title (and thus no `: \s`part after the ep ) in the movie name so the regex doesnt match
This
```yaml
                - regex: (.+)(?:\sEp\d).*
```
should be enough